src/api/watch.jl

```diff
@@ -57,6 +57,12 @@ function watchloop(
                 break
             end
 
+            if "node" in keys(resp)
```

Comment:
I'd probably write this with 1 if-block as `if "node in keys(resp) && "modifiedIndex" in keys(resp["node"])`. It might be good to add a comment on why we need to do `wait_index = resp["node"]["modifiedIndex"]`.

Reply:
sure. in a bit